packages/aws-cdk-lib/aws-sns/lib/topic.ts

```diff
@@ -226,6 +238,7 @@ export class Topic extends TopicBase {
       kmsMasterKeyId: props.masterKey && props.masterKey.keyArn,
       contentBasedDeduplication: props.contentBasedDeduplication,
       fifoTopic: props.fifo,
+      signatureVersion: props.signatureVersion,
```

Comment:
Can we add validation for supported signature versions (1 and 2) as specified in the [docs](https://docs.aws.amazon.com/sns/latest/dg/sns-verify-signature-of-message.html)?